R/linters.R

```diff
@@ -169,6 +149,8 @@ addLinter("filepath.capitalization", linter(
       lapply(regex, function(x) {
 
         which(
+          # Only examine paths containing a slash to reduce false positives
```

Comment:
The side-effect, unfortunately, is that a file named `MyData.csv` but referenced as `mydata.csv` will go uncaught. So sad.

Reply:
Yeah, it's a bummer. I have racked my brains to come up with another technique to use, but I have failed so far.

Reply:
An option that comes to mind for me is to adjust some of the file-reading utilities; if they experience a file-not-found error, they could look for a case-insensitive same-name file and add that context to the error message.

> File `mydata.csv` is not found, but there is a file named `MyData.csv`.

This feels excessive in most cases, though.. and we wouldn't be able to adjust all such functions.